ee/clickhouse/views/test/test_clickhouse_path_person.py

```diff
@@ -18,13 +18,15 @@ def _create_sample_data(self, num, delete=False):
             if delete:
                 person = Person.objects.create(distinct_ids=[f"user_{i}"], team=self.team)
```

Comment:
Why are all these person changes required

Reply:
Because the persons are deleted, `_create_event` tries to find an unknown person, and when it fails, assigns a random UUID to each event.

Thus, even if we generated 20 persons in the test, there's 50 person IDs generated, which was completely borking the test.

This should've failed on the lost persons PR, but didn't (but did fail on master, so not sure what happened)

Reply:
(only matters for person-on-events)